src/Views/Form/Templates/Sequoia/assets/css/form.scss

```diff
@@ -1319,6 +1319,11 @@ form.give-form .form-row select.multiselect {
 	}
 }
 
+/* Hide everything which is not output by Give core or addon */
+body > *:not([class^='give']):not([id^='give']) {
```

Comment:
@henryholtgeerts I really need your opinion on whether this is a good idea or not. From the look of it I could see conflicts arising with our internal add-ons if their classes are not properly prefixed with "give".

Reply:
I find out that we do not print anything outside of donation form wrapper.
@henryholtgeerts opinion will be valuable.

![image](https://user-images.githubusercontent.com/1784821/86037964-cf3fa100-ba5d-11ea-953b-5b4c718e87a0.png)

Reply:
@ravinderk Is right, we are not adding any markup outside the form wrap div. I think we should only target div outside of that wrapper, or else we run the serious (almost certain) risk of hiding elements from various addons which are not properly prefixed.

Reply:
@henryholtgeerts Yes you are right and for precaution, I am targeting only direct child of `body` tag.